app/src/main/java/com/example/messengerAndroid/ui/MyContactsActivity.kt

```diff
@@ -54,7 +52,14 @@ class MyContactsActivity : AppCompatActivity() {
         binding.textViewAddContacts.setOnClickListener {
```

Comment:
In setListeners() have to be only method calls but not their implementation.

Reply:
As for me, it is ok to have some code in the setListeners but it would be better to decompose the code to separate functions in order to make the functionality more readable.